vignettes/carbon_tracking.Rmd

```diff
@@ -75,13 +77,16 @@ each model time point after `trackingDate`.
 ``` {r hector.tracking}
 tdata <- get_tracking_data(core)
 head(tdata)
-
-# TODO - the Diff entries are unneeded debugging information; remove
-tdata <- subset(tdata, pool_name != "Diff")
 ```
 
-The `tdata` dataset now contains information on year, pool name, amount of 
-carbon in each pool, and the fraction of each source for that pool.
+The `tdata` dataset now contains information on year, the model component,
+the name of the pool within that component, the amount of 
+carbon in the pool, and the fraction (0-1) of each source for that pool. 
+The source fractions for a given pool and year will always sum to 1.
+
+Side note: the names in the carbon tracking data (`pool_name`, `pool_value`,
```

Comment:
Isn't 'component' also a name returned from the model output?

Reply:
Yes; I'm just talking about the three that overlap with the model output but differ in their names.

Reply:
Clarifying